src/compiler/utilities.ts

```diff
@@ -1220,6 +1220,11 @@ namespace ts {
             if (isSourceOfAssignmentExpressionStatement) {
                 return node.parent.parent.jsDocComment;
             }
+
+            const isPropertyAssignmentExpression = node.parent && node.parent.kind === SyntaxKind.PropertyAssignment;

```

Comment:
`parent` gets used quite a lot here and above - consider declaring a `parent` constant?